services/ui/src/js/components/DiagnosticView/ScalarVariable.js

```diff
@@ -14,7 +14,7 @@ export default function ScalarVariable(props) {
   const guess = data.map((feature) => feature.properties.guess);
   const analysis = data.map((feature) => feature.properties.analysis);
 
-  return html`<h2 data-colspan="full">Guess</h2>
+  return html`<h2 class="font-ui-lg text-bold grid-col-full">Guess</h2>
```

Comment:
Not knowing much about css - I wasn't able to track down a `data-colspan` element in MDN. Was this defined by the project's styles in the old `_layout.scss` here? If so, I'm guessing you needed to remove it here since it was refactored out of `_layout.scss`?

https://github.com/NOAA-GSL/unified-graphics/blob/f48256d9bb2ee86a65e77bfa11aad1da2d0c8f03/services/ui/src/styles/_layout.scss#L24-L26

Reply:
Never mind! You covered this in 0774a8d61355bcb7023ef30221ba74ffc38aaebc. Thanks for the commit message!

Reply:
Data attributes are kind of an HTML extension mechanism. You’re allowed to add arbitrary attributes to any HTML element as long as it starts with `data-`. They’re also made available through the DOM via the `datalist` property.

It’s become a lot more common to use HTML attributes as selectors in CSS instead of using classes exclusively, but since most of our utility classes are actual CSS classes, it made sense to me to switch this one back to a class.

https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

Reply:
Ahhh... that's great to know. It's interesting how the dashes are converted to CamelCase as well. Thanks for linking in the docs!